.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
tu się eslinty zduplikowały chyba

Reply:
ja to robie wg instrukcji od cb... nie wiem co tam sie wydarzylo

Reply:
usunąć ten .cjs możliwe że w vite w template to jest